ovs/lib/tests/vdisk_tests/test_get_set_config_params.py

```diff
@@ -49,7 +49,7 @@ def setUpClass(cls):
         fakesleep.monkey_patch()
         EtcdConfiguration.set('/ovs/framework/storagedriver|mds_tlogs', 100)
         EtcdConfiguration.set('/ovs/framework/storagedriver|mds_maxload', 75)
-        EtcdConfiguration.set('/ovs/framework/storagedriver|mds_safety', 2)
+        EtcdConfiguration.set('/ovs/framework/storagedriver|mds_safety', 3)
```

Comment:
Valid point. Test will indeed have to be reviewed. The creation of a lot of disks will fail already as the mds service isn't available on 3 nodes. Maybe better to also extend the test to check if mds safety is as the one set is actually achieved.